includes/REST_API/Autosaves_Controller.php

```diff
@@ -172,7 +172,7 @@ public function register_routes() {
 	 *
 	 * @return WP_REST_Response Response object.
 	 */
-	public function prepare_item_for_response( $post, $request ) {
+	public function prepare_item_for_response( $post, $request ): WP_REST_Response {
```

Comment:
This has a filter at the end of it, meaning it should return an WP_Error or something else. I didn't add this on purpose.

Reply:
Hmm. Then we should change the `@return` to `mixed`?

Reply:
No, that is what is recommended, it is possible for it be changed and it should break things.

Reply:
Or I just revert the changes I suppose 🤷